scipy/io/mmio.py

```diff
@@ -505,7 +532,7 @@ def _parse_body(self, stream):
         return a
 
     #---------------------------------------------------------------------------
-    def _write(self, stream, a, comment='', field=None, precision=None):
+    def _write(self, stream, a, comment='', field=None, precision=None, symmetry=None):
```

Comment:
In most cases, a symmetry check for non-symmetric matrices should not take soo long, since only one non-symmetric entry must be found. Nevertheless, it might be usefully to set the `general` format as default and only determine the symmetry type if `symmetry` is explicitly set to None for performance reasons. Do you agree?